roles/rsyslog/tasks/deploy.yaml

```diff
@@ -1,20 +1,23 @@
 ---
 - name: Install/Update required packages
   package:
-    name: "{{ rsyslog_role_packages }}"
+    name: "{{ __rsyslog_packages }}"
     state: latest
```

Comment:
I propose to use `present`, not `latest`. Most of our roles use this, so let's be consistent. @robled @greg-hellings @seandst do you have an opinion here?
If we want to be able to use `latest`, we can introduce a _xxx_`_packages_state` variable, like here:

https://github.com/geerlingguy/ansible-role-apache/blob/77f39119bc12fd839d16a55a19354d5c159041fa/defaults/main.yml#L58

Reply:
Leaving the state as latest to make sure the newer packages are installed/updated if available.
Leaving the conversation opened.